### DISCUSSION TITLE
Is toss() mandatory to run the tests ?

### DISCUSSION THREAD

**Discussion by mihirsasmal** [pactumjs/pactum#241] [Q&A] [answered]
Hi, 

I am able to run the tests without using` toss()` like this :- 
```
const response = await spec().get('url').expectStatus(200);
const idForFutureUse = response.body.id;

```
So I am wondering why I should use `toss ()`?

**Comment by ASaiAnudeep** [comment, marked as answer]
`toss` is not mandatory if you are using await statement. Typescript users will use it to avoid lint issues.
